### ISSUE TITLE
Ability to curate messages

### ISSUE THREAD

**Issue by pokeball99** [slatinsky/DiscordChatExporter-frontend#8]
this may be out of scope, but sometimes there are a bunch of messages in a channel that you don't want to see/preserve(15k posts that are just variants of "hey/morning/goodnight")

some way of using this(instead of manually editing json/deleting images) to curate a channel would be helpful

i guess this would be best done by having it make a copy before you modify it, and/or leaving it up to the user if they want to make a backup or if they want to edit the "master" archive.

**Comment by slatinsky** [comment]
Hello,

After the client-server rewrite is complete, you will be able to host the viewer over the internet. 
(you can do it now too if your exports are small) 

My policy with the DiscordChatExporter-frontend is to NEVER modify the original exports. Exports folder stays read only.
Currently all pre computed data is saved in different folder (dcef/backend/preprocess/temp) and exports folder is used only to "hotlink" assets. 

I was planning to add feature to hide messages based on author ids (if you host it over the internet and some user doesn't like it), but I don't see the use to add more filters like you describe. 

Once the above feature is done, your change (to hide messages based on content) will be easy to implement with a little source code modification, but it seems to be outside the scope of the main project and won't be implemented.

**Comment by slatinsky** [comment]
Hello,

Ability to select which users or discord servers to show was added to [v2.8.0 release](https://github.com/slatinsky/DiscordChatExporter-frontend/releases/tag/v2.8.0). This change doesn't modify the original JSON exports.

closing as completed
